Packs/ProofpointEmailSecurity/Integrations/ProofpointEmailSecurityEventCollector/ProofpointEmailSecurityEventCollector.yml

```diff
@@ -44,13 +44,24 @@ configuration:
   type: 8
   required: false
   section: Collect
+- additionalinfo: Denotes which event type to fetch, if not provided will fetch all kinds.
```

Comment:
isnt there a way to provide a default? (which also adds to the description automatically i think)

Reply:
handled in the code.

Reply:
yes but best to also make the default available in the yml for documentation reasons. Needs to be in the code also for upgrade purposes